### PR TITLE
feat: [SRM-14418]: Annotations enhances and fixes

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Removing old logic of showing downtime icons on the chart.
2. Fixing any UX issues raised by @yuvrajharness - https://docs.google.com/document/d/1fg6w8mkQwoglbnpjunpJJT6HR0aOZPOKphMlo_FnWxs/edit#
3. Adding logic for Account level CRUD operations for Annotations
- Delete account level annotation
- Save account level annotation
- Update account level annotation
4. https://harness.atlassian.net/browse/SRM-14420
5. New Design suggested by Thani for the scenario where multiple widgets have same start time.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by harshilbits** [comment]
retrigger titlecheck

File: src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/TimelineRow.tsx [hunk 1]
```diff
@@ -47,14 +47,14 @@ export function TimelineRow(props: TimelineRowProps): JSX.Element {
     const containerWidth = (timelineRowRef.current.parentElement?.getBoundingClientRect().width || 0) - leftOffset
     setDataWithPositions(getDataWithPositions(containerWidth, startTimestamp, endTimestamp, data))
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [timelineRowRef?.current, data, endTimestamp, startTimestamp])
+  }, [timelineRowRef?.current, data, endTimestamp, startTimestamp, leftOffset])
+
+  // [data, endTimestamp, startTimestamp, leftOffset]
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/TimelineRow.tsx:52]: remove commented code

File: src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/components/WidgetsWithSameStartTime/WidgetsWithSameStartTime.tsx [hunk 2]
```diff
@@ -127,116 +139,107 @@ export default function WidgetsWithSameStartTime(props: WidgetsWithSameStartTime
...
-                                    </Text>
-                                    <hr className={css.division} />
-                                  </Layout.Vertical>
-                                )
-                              })
-                            : null}
-                        </>
-                      )}
+                <Container className={css.annotationContainer}>
+                  {loading ? (
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/components/WidgetsWithSameStartTime/WidgetsWithSameStartTime.tsx:148]: We will need to refactor this JSX into smaller components for better readability

**Review by deepesh-ui — APPROVED** [review]

File: src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/components/WidgetsWithSameStartTime/WidgetsWithSameStartTime.tsx (hunk 2 above)
Comment (praneshg239) [reply src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/components/WidgetsWithSameStartTime/WidgetsWithSameStartTime.tsx:148]: Agree.

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/components/WidgetsWithSameStartTime/WidgetsWithSameStartTime.tsx (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/components/ChangeTimeline/components/TimelineRow/components/WidgetsWithSameStartTime/WidgetsWithSameStartTime.tsx:148]: Sure , will check on this.

**Pull request merged**